lib/manager/jenkins/readme.md

```diff
@@ -1,13 +1,4 @@
-The Jenkins manager supports the following format of the plugin list:
+The Jenkins manager supports a custom text or YAML format of the plugin list as described [here](https://github.com/jenkinsci/plugin-installation-manager-tool#plugin-input-format).
+Currently, only versions from the main [update center](https://updates.jenkins.io/) are supported.
```

Comment:
```suggestion
Only versions from the main [update center](https://updates.jenkins.io/) are supported.
```

I think this is a bit cleaner, as it doesn't imply that other "update centers" will be supported later on.

If support for other "update centers" is added, this text can be updated to list the supported centers.